app/src/main/java/org/oppia/android/app/devoptions/DeveloperOptionsActivityPresenter.kt

```diff
@@ -55,7 +55,11 @@ class DeveloperOptionsActivityPresenter @Inject constructor(
       ) as DeveloperOptionsFragment?
   }
 
+  /**
+   * Called when the 'force crash' button is clicked by the user.
+   * It throws a RuntimeException which results in crashing of the app.
```

Comment:
First, KDocs should fill up as much space as they can before wrapping (at the 100 character line limit). Second, maybe instead of specifying a really implementation-specific detail like 'throws a RuntimeException' we could instead explain what's conceptually happening, e.g.:

``This function crashes the app and will not return.``

In fact, you could even have it return Nothing since that's the designated Kotlin type for this exact circumstance.

Reply:
Updated KDoc. Also, added `Nothing` as the return type of the function.